Pod/Classes/CMHUser.h

```diff
@@ -84,8 +84,14 @@ typedef void(^CMHFetchConsentCompletion)(CMHConsent *_Nullable consent, NSError
  *  Fails with an error if the current user is not logged in. Calback returns `nil`
  *  if no consent exists for the given user and study descriptor.
  *
- *  @warning The behavior of this method if the current user is logged in should
- *  be considered undefined. Log the current user out before calling this method.
+ *  @warning Common sense suggests each participant would have one consent for each
```

Comment:
Can this read something like "The expected use case is that each participant..."? "Common sense suggests" could come off as condescending for unthought but useful or required cases.